README.md

```diff
@@ -1,5 +1,21 @@
 # meta-xt-prod-devel-rcar #
 
+# Table of contents
```

Comment:
This is a good change, but it should go into a separate commit

Reply:
Hi @lorc, done.